### ISSUE TITLE
Use this in Android

### ISSUE THREAD

**Issue by zhouzhuli** [Unity-Technologies/NativeAudioPlugins#3]
I want to use these effects on Android, but this can  be used only on the computer！

**Comment by hargisss** [comment]
Here here!

**Comment by Nemisindo** [comment]
Hi can you guys make a tutorial on how to export plugins for android?

**Comment by AlexxUnity** [comment]
Yeah the [documentation](https://docs.unity3d.com/Manual/android-native-plugins-create.html) around this is pretty lacking and basically says "get the Android NDK and work it out"

We are looking into providing better documentation and build system around this, thanks for raising it (and sorry for the radio silence).

**Comment by gregbeller** [comment]
+1. That would be awesome... Any progress on that part? For now, an embed release in the bundle would make the job. Those plugins are awesome to be used out of the box... Thanks BTW...

**Comment by korinVR** [comment]
I added Android build to this fork, but the built library is not recognized.

https://github.com/korinVR/NativeAudioPlugins/tree/android

![Image](https://github.com/user-attachments/assets/1735815f-9f10-4472-8ac2-c69f8ea99bbc)

I'm probably missing something very obvious... Any information would be appreciated.

**Comment by desmondberg** [comment]
Any update on docs for writing native audio plugins for Android?
